<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="FX-JSJ800" name="com.zhou.autotest.M1_fun_tv_home_user4" tests="5" failures="0" timestamp="11 Mar 2016 02:02:04 GMT" time="75.362" errors="1">
  <testcase name="testCookie" time="32.170" classname="com.zhou.autotest.M1_fun_tv_home_user4"/>
  <testcase name="testVplay" time="16.192" classname="com.zhou.autotest.M1_fun_tv_home_user4"/>
  <testcase name="testNextAndLocalStorage" time="10.146" classname="com.zhou.autotest.M1_fun_tv_home_user4">
    <error type="java.lang.AssertionError" message="local storage : assert url ,vtitle expected:&lt;true&gt; but was:&lt;false&gt;">
      <![CDATA[java.lang.AssertionError: local storage : assert url ,vtitle expected:<true> but was:<false>
	at org.testng.Assert.fail(Assert.java:89)
	at org.testng.Assert.failNotEquals(Assert.java:480)
	at org.testng.Assert.assertTrue(Assert.java:37)
	at com.zhou.autotest.M1_fun_tv_home_common.LocalStorage(M1_fun_tv_home_common.java:209)
	at com.zhou.autotest.M1_fun_tv_home_user4.testNextAndLocalStorage(M1_fun_tv_home_user4.java:120)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:76)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at com.zhou.run.Run.main(Run.java:73)
]]>
    </error>
  </testcase>
  <testcase name="testClickSkip" time="13.809" classname="com.zhou.autotest.M1_fun_tv_home_user4"/>
  <testcase name="testLocalStorage" time="3.045" classname="com.zhou.autotest.M1_fun_tv_home_user4"/>
</testsuite>
